README.md

```diff
@@ -8,6 +8,10 @@ where the core framework accomodates for asynchronous calls and even retries
 if appropriate.
 
 
+# Setup
+pip install -r requirements.txt
```

Comment:
Should we add something about setting up a `virtualenv`, or are we not worrying about dev environments in this scenario?